hack/build-in-docker.sh

```diff
@@ -7,6 +7,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function absolute_path() { pushd . > /dev/null; if [ -d "$1" ]; then cd "$1"; dirs -l +0; else cd "`dirname \"$1\"`"; cur_dir=`dirs -l +0`; if [ "$cur_dir" == "/" ]; then echo "$cur_dir`basename \"$1\"`"; else echo "$cur_dir/`basename \"$1\"`"; fi; fi; popd > /dev/null; }
```

Comment:
@smarterclayton so I found this and they claim it should work on **all** platforms ;-)

it works on my arch linux, can you try osx?